rotkehlchen/db/snapshots.py

```diff
@@ -297,13 +314,34 @@ def _import_snapshot(
                     usd_value=str(FVal(entry['usd_value'])),
                 ),
             )
+
         try:
             self.db.add_multiple_balances(processed_balances_list)
             self.db.add_multiple_location_data(processed_location_data_list)
         except InputError as err:
             return False, str(err)
         return True, ''
 
+    def update(
+        self,
+        timestamp: Timestamp,
+        balances_snapshot: List[DBAssetBalance],
+        location_data_snapshot: List[LocationData],
+    ) -> Tuple[bool, str]:
+        """Updates a snapshot of the database at a given timestamp."""
+        # delete the existing snapshot of that timestamp
+        is_success, message = self.delete(timestamp)
+        if is_success is False:
+            return is_success, message
+
+        # update the snapshot with the provided data.
+        try:
+            self.db.add_multiple_balances(balances_snapshot)
+            self.db.add_multiple_location_data(location_data_snapshot)
+        except InputError as err:
+            return False, str(err)
+        return True, 'Snapshot updated succesfully.'
```

Comment:
Why do you return a message here, is it necessary? As I see from the codebase, we usually return empty message if everything went OK.